www/__mocks__/cordovaMocks.ts

```diff
@@ -100,11 +101,20 @@ export const mockBEMUserCache = () => {
       );
     },
     getDocument: (key: string, withMetadata?: boolean) => {
-      return new Promise<any[]>((rs, rj) =>
-        setTimeout(() => {
-          rs(_storage[key]);
-        }, 100),
-      );
+      //returns the config provided as a paramenter to this mock!
+      if (key == 'config/app_ui_config') {
+        return new Promise<any>((rs, rj) =>
+          setTimeout(() => {
+            rs(config);
```

Comment:
I went ahead and added the fakeConfig as the fallback, that way it can be in place and only need to be passed in if you want to test special cases (as I was wanting to do with enketoHelper)